src/pages/coaches/roy-derks.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Roy Derks
+title: Senior Frontend Developer at
+companyName: City of Amsterdam
+companyLink: https://data.amsterdam.nl
+gitHub: https://github.com/royderks
+medium: https://medium.com/hackteam/
+twitter: https://twitter.com/gethackteam
+linkedIn: https://uk.linkedin.com/in/royderks1
+blockquote: When in doubt, start coding.
+imageDescription: React GraphQL Academy coach Roy Derks, speaking at a developer conference with microphone.
+imageSrc: https://firebasestorage.googleapis.com/v0/b/reactjsacademy-react.appspot.com/o/team%2Froy.jpg?alt=media
+tags: []
+---
+
+<ul>
```

Comment:
@ric9176 why don't we use markdown for the list? https://www.markdownguide.org/basic-syntax/#unordered-lists